packages/@react-stately/color/src/Color.ts

```diff
@@ -128,6 +138,82 @@ class RGBColor extends Color {
     return this.red << 16 | this.green << 8 | this.blue;
   }
 
+  private toHSB(): Color {
+    let r = this.red / 255;
+    let g = this.green / 255;
+    let b = this.blue / 255;
+    let max = Math.max(r, g, b);
+    let min = Math.min(r, g, b);
+    let h: number;
+    let s: number;
+    let d = max - min;
+
+    s = max === 0 ? 0 : d / max;
+
+    if (d === 0) {
+      h = 0; // achromatic
+    } else {
+      switch (max) {
+        case r:
+          h = (g - b) / d + (g < b ? 6 : 0);
+          break;
+        case g:
+          h = (b - r) / d + 2;
+          break;
+        case b:
+          h = (r - g) / d + 4;
+          break;
+      }
+
+      h /= 6;
+    }
+
+    return new HSBColor(
+      h * 360,
+      s * 100,
+      b * 100,
```

Comment:
This doesn't seem right. Variable `b` is defined as `this.blue / 255` at [R144](https://github.com/adobe/react-spectrum/pull/2496/files?authenticity_token=gefkXkUI6TFuXRPxyiIX1fgOQDifVT8JQI4tlVA8Rp8DGmeNbj43vXIMLAsc%2Ff5%2BP1rkqD%2B3ut%2Bttg6qKxGvqA%3D%3D&file-filters%5B%5D=.js&file-filters%5B%5D=.lock&file-filters%5B%5D=.ts&file-filters%5B%5D=.txt#diff-2737484a486e2634802c43f60406fc1231698480673f4268bd1e8f029448bcccR144).

Try this:
```ts
/**
   * Converts an RGB color value to HSB.
   * Conversion formula adapted from https://en.wikipedia.org/wiki/HSL_and_HSV#From_RGB.
   * @returns An HSBColor object.
   */
  private toHSB(): Color {
    const red = this.red / 255;
    const green = this.green / 255;
    const blue = this.blue / 255;
    const min = Math.min(red, green, blue);
    const brightness = Math.max(red, green, blue);
    const chroma = brightness - min;
    const saturation = brightness === 0 ? 0 : chroma / brightness;
    let hue = 0; // achromatic  

    if (chroma !== 0) {
      switch (brightness) {
        case red:
          hue = (green - blue) / chroma + (green < blue ? 6 : 0);
          break;
        case green:
          hue = (blue - red) / chroma + 2;
          break;
        case blue:
          hue = (red - green) / chroma + 4;
          break;
      }
  
      hue /= 6;
    }

    return new HSBColor(
      hue * 360,
      saturation * 100,
      brightness * 100,
      this.alpha
    );
  }
```